pom.xml

```diff
@@ -58,6 +58,18 @@
             <snapshots>
                 <enabled>false</enabled>
             </snapshots>
+            <releases>
+                <enabled>false</enabled>
```

Comment:
OK, so we are blacklisting it explicitly here in order to make sure D-Net transitive dependencies are not retrieved from an old repo, right?

So it is a temporary solution until the repo is resolved in DNet part of the project we are relying on.

I would introduce just a minor change: since the new repo is the one which should be used I would drop the `-new` suffix (indicating this repo as the main one) and add an `-old` suffix (or `-deprecated`?, something indicating this is obsolete) to the old repo to indicate this is the one to be removed. So the 2nd phase of the repo transition will be simple `dnet45-releases-old` repo removal from the `pom.xml` file.

Reply:
Due to updates this part of POM was changed and no new/old/deprecated repos are needed. I'm commenting on the solution in the main conversation.